tests/integration/conftest.py

```diff
@@ -84,7 +84,7 @@ def convert2rhel(shell):
     @contextmanager
     def factory(
         options: str,
-        timeout: int = 30 * 60,
+        timeout: int = 60 * 60,
```

Comment:
When I was running the tests locally it always timeouts after 30minutes. The conversion is much slower than using activation key or regular RHSM. Maybe we should investigate more?

(Increasing timeout ends in successful conversion)